test/e2e/generated/BUILD

```diff
@@ -12,6 +12,9 @@ load(
 
 go_library(
     name = "go_default_library",
-    srcs = ["main.go"],
+    srcs = [
+        "bindata.go",
```

Comment:
This wouldn't have passed verify on a clean workspace since this file is not checked in. This was fixed but then reverted, so I'm trying to get it unreverted